sound/soc/sof/ipc4-topology.c

```diff
@@ -1103,6 +1103,16 @@ sof_ipc4_update_resource_usage(struct snd_sof_dev *sdev, struct snd_sof_widget *
 	/* Update the base_config->cpc from the module manifest - max CPC value */
 	sof_ipc4_update_cpc_from_manifest(sdev, swidget->module_info, base_config, false);
 
+	if (!base_config->cpc) {
+		struct sof_ipc4_fw_data *ipc4_data = sdev->private;
+
+		base_config->cpc = ipc4_data->max_core_frequency;
```

Comment:
`base_config->cpc = ipc4_data->max_core_frequency / MSEC_PER_SEC;` or `/ 1000` as we want 1ms chunk number.